RoddyCore/src/de/dkfz/roddy/config/loader/ProcessingToolReader.groovy

```diff
@@ -234,60 +221,68 @@ class ProcessingToolReader {
      * @return
      */
     ToolEntry.ToolParameter parseToolParameter(NodeChild child, String toolID) {
-        String type = readAttribute(child, "type")
-        if (type == "file") { //Load a file
+        String type = readAttribute(child, 'type')
+        if (type == 'file') { // Load a file
             return parseFile(child, toolID)
-        } else if (type == "tuple") {
+        } else if (type == 'tuple') {
             return parseTuple(child, toolID)
-        } else if (type == "filegroup") {
+        } else if (type == 'filegroup') {
             return parseFileGroup(child, toolID)
         } else if (type == CVALUE_TYPE_STRING) {
-            ToolStringParameter.ParameterSetbyOptions setby = Enum.valueOf(ToolStringParameter.ParameterSetbyOptions.class, extractAttributeText(child, "setby", ToolStringParameter.ParameterSetbyOptions.callingCode.name()))
+            ToolStringParameter.ParameterSetbyOptions setby =
+                    Enum.valueOf(ToolStringParameter.ParameterSetbyOptions.class,
+                            extractAttributeText(child, 'setby',
+                                    ToolStringParameter.ParameterSetbyOptions.callingCode.name()))
```

Comment:
Also here (see below), consider placing the closing braces on new lines to help Idea with formatting.